examples/hal/quad/main.rs

```diff
@@ -176,6 +177,7 @@ fn main() {
             colors: &[(0, i::Layout::ColorAttachmentOptimal)],
             depth_stencil: None,
             inputs: &[],
+            resolves: &[],
```

Comment:
We should probably have `Default` implemented for `pass::SubpassDesc`, so that cases like this can benefit from only supplying a few fields